common/fe-ste-models.go

```diff
@@ -1102,7 +1102,7 @@ func (i *InvalidMetadataHandleOption) UnmarshalJSON(b []byte) error {
 // //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 const (
 	DefaultBlockBlobBlockSize      = 8 * 1024 * 1024
-	MaxBlockBlobBlockSize          = 4000 * 1024 * 1024
+	MaxBlockBlobBlockSize          = 5000 * 1024 * 1024
```

Comment:
We can't update the BlockSize since the limit is 4000MB

https://learn.microsoft.com/en-us/rest/api/storageservices/put-block?tabs=microsoft-entra-id#remarks.

We need the code that is failing to understand the difference between a blob being created with a PutBlob vs a StageBlock/CommitBlockList path and check the appropriate sizes